src/Altinn.Platform/Altinn.Platform.Register/Register/Core/PersonLookupService.cs

```diff
@@ -0,0 +1,79 @@
+#nullable enable
+using System;
+using System.Threading.Tasks;
+
+using Altinn.Platform.Register.Models;
+using Altinn.Platform.Register.Services.Interfaces;
+
+using Microsoft.Extensions.Caching.Memory;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+
+namespace Altinn.Platform.Register.Core
+{
+    /// <summary>
+    /// Represents the business logic related to checking if a national identity number is in use.
+    /// </summary>
+    public class PersonLookupService : IPersonLookup
+    {
+        private const string PersonLookupFailedAttempts = "Person-Lookup-Failed-Attempts";
```

Comment:
No biggie and more a question, but do we have an "ALL_UPPERCASE" policy for constants? I'm not sure if we do, but I'll admit I prefer it to make it more visible.

Reply:
I would use all uppercase for public constants, but PascalCasing for private constants.